libs/net/services/Interfaces/IApiService.cs

```diff
@@ -7,6 +7,13 @@ namespace TNO.Services;
 /// </summary>
 public interface IApiService
 {
+    #region Properties
+    /// <summary>
+    /// get - HTTP client to make HTTP requests.
+    /// </summary>
+    IOpenIdConnectRequestClient OpenClient { get; }
```

Comment:
Made public so that inherited classes have access to change the request default headers.